src/well-known-headers.json

```diff
@@ -12,6 +12,7 @@
   "access-control",
   "access-control-request-headers",
   "access-control-request-method",
+  "activate-storage-access",
```

Comment:
this one is response header
https://developers.google.com/privacy-sandbox/blog/storage-access-api-headers-logic#:~:text=The%20Activate%2DStorage%2DAccess%20header,cookies%20for%20the%20requested%20resource.